src/main/java/com/dnastack/wes/cromwell/CromwellService.java

```diff
@@ -61,14 +66,16 @@ public class CromwellService {
         PathTranslatorFactory pathTranslatorFactory,
         CromwellWesMapper cromwellWesMapper,
         AppConfig appConfig,
-        CromwellConfig config
+        CromwellConfig config,
+        @Qualifier("defaultAsyncOperationExecutor") ThreadPoolTaskExecutor executor
```

Comment:
A few tidbits:
- In most cases its better to reference the interface or abstract class so that you can change the implementation at run time if you do not need the features of the concrete class. In this case the `@Bean` could return `TaskExecutor` and this above could be `TaskExecutor`
- In Spring there is an implicit qualifier, the property name. If you changed the name of this to `TaskExecutor defaultAsyncOperationExecutor` it would still inject the same bean you are looking for

Reply:
This makes sense, thank you! I updated to use `TaskExecutor` and the `defaultAsyncOperationExecutor` name